nemo_skills/evaluation/metrics/speechlm_metrics.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from nemo_skills.evaluation.metrics.base import BaseMetrics, as_int, as_percentage
+from nemo_skills.utils import get_logger_name
+
+LOG = logging.getLogger(get_logger_name(__file__))
+
+
+class SpeechLMMetrics(BaseMetrics):
```

Comment:
Should we have a more specific name here? I imagine not all speech lm benchmarks will use this.

Also you don't have to use a single class for all 3 subtasks in mmau, if it's more convenient, we can have 3 different metric classes for them, but up to you